### PR TITLE
Create new design for subscribe email form

### PR DESCRIPTION
**Pull request by davydovanton**
Updated #2820 
## Screenshots
screenshot 2016-02-07 21 42 09
screenshot 2016-02-07 21 42 19

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -5,8 +5,12 @@
         class="btn btn-default">info@contribsys.com</a></h4>
   </div>
   <form class="text-center" action="https://tinyletter.com/sidekiq" method="post" target="popupwindow" onsubmit="window.open('https://tinyletter.com/sidekiq', 'popupwindow', 'scrollbars=yes,width=800,height=600');return true">
-    <label for="tlemail">Subscribe to our semi-occasional newsletter and keep up to date on new Sidekiq features and changes.</label></p>
-    <input type="text" style="width:240px" name="email" id="tlemail" /><input type="hidden" value="1" name="embed"/><input type="submit" value="Subscribe" /></p>
+    <label for="tlemail">Subscribe to our semi-occasional newsletter and keep up to date on new Sidekiq features and changes.</label>
+    <div class="sibscribe-form">
```
Comment (mperham) [inline _includes/footer.html]: Oops, misspelling here: subscribe

Comment (davydovanton) [inline _includes/footer.html]: oh, thanks for catch :+1:

**Comment by davydovanton** [comment]
@mperham done.

**Comment by davydovanton** [comment]
thanks for merge :star2:

**Comment by mperham** [comment]
Thanks for the polish!

**Pull request merged**